### ISSUE TITLE
Built-in Flow (0.7.0)

### ISSUE THREAD

**Issue by svipas** [flowtype/flow-for-vscode#135]
As far as I understand in 0.7.0 version if you don't have Flow it will be used from the extension? If yes, it doesn't working for me, I get error: `[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g`.

VS Code: 1.12.2
Linux Ubuntu 16.04.

**Comment by jsslai** [comment]
I can't use it either. Seems it tries to find Flow from path which doesn't exist: 
`$HOME/.vscode/extensions/flowtype.flow-for-vscode-0.7.0/node_modules/.bin/flow`

**Comment by showmeyourhits** [comment]
Can someone share `flow.pathToFlow`settings parameter that will point to the current project `node_modules` folder?
I tried something like:
`"flow.pathToFlow": "${rootPath}/node_modules/.bin/flow"`
but it isn't working.

**Comment by silvenon** [comment]
@showmeyourhits `"flow.useNPMPackagedFlow": true`?

**Comment by 7ynk3r** [comment]
1. make sure that you don't have `flow-bin` global, but as a dev dependency in your package.
2. in you vs code settings, use the following
```
  "flow.enabled": true,
  "flow.runOnEdit": true,
  "flow.showStatus": true,
  "flow.runOnAllFiles": true,
  "flow.stopFlowOnExit": true,
  "flow.useNPMPackagedFlow": true,
  "javascript.validate.enable": false,
```
3. restart vs code.

**Comment by camsloanftc** [comment]
Thanks 7ynk3r. Been trying to find this for a bit now.

**Comment by chrisblossom** [comment]
I'm seeing this error every time I load a project that does not use flow. Is there a way to disable this when flow is not found in the project? I am using ``"flow.useNPMPackagedFlow": true`` and do not have ``flow-bin`` as a global.

**Comment by rob10e** [comment]
My 2 cents: I was having the same problem. I had just installed a fresh copy of Ubuntu 18.04, and immediately installed NVM, loading the latest node stable version. Everything else was working great, except Flow. After digging into the developer tools for VSCode, I realized it was trying to launch flow with the default node location and not the location created by NVM. As there was no version of node installed there, it couldn't launch Flow. Solution: I installed node through the Ubuntu repository. Everything is now fixed.
